src/dataset-builder/DatasetBuilderUtils.test.ts

```diff
@@ -224,9 +224,9 @@ describe('test HighlightConceptName', () => {
     const conceptName = 'Clinical Finding';
 
     const result = div({ style: { display: 'flex' } }, [
-      div(['Clinical ']),
-      div({ style: { fontWeight: 600 } }, ['Finding']),
-      div(['']),
+      div({ style: { whiteSpace: 'pre' } }, ['Clinical ']),
+      div({ style: { fontWeight: 600, whiteSpace: 'pre' } }, ['Finding']),
+      div({ style: { whiteSpace: 'pre' } }, ['']),
```

Comment:
Is there a way to refactor these three cases to avoid the duplication here? Or maybe write a test that doesn't rely on the exact formatting details?